### PR TITLE
Update PostCSS to a maintained version

### PR DESCRIPTION
**Pull request by MKodde**
To get a working set of dependencies, other packages where also upgraded. Some noteworthy are:

 - Webpack Encore
 - Node-sass
 - TS Loader

In order to run the newer webpack version, we are required to run a younger Node version. We jumped from Node 8 to Node 14 (LTS).

### CODE REVIEW

File: ci/docker/php-fpm/Dockerfile [hunk 1]
```diff
@@ -5,12 +5,15 @@ COPY docker/php-fpm/app.ini /usr/local/etc/php/conf.d/
 # Copy the app_test.php.dist to web/app.php
 COPY index.php ../public/index.php
 
+# Yank the node and npm binaries from the official Node docker container
+COPY --from=node:14 /usr/local/lib/node_modules /usr/local/lib/node_modules
+COPY --from=node:14 /usr/local/bin/node /usr/local/bin/node
+RUN ln -s /usr/local/lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm
+RUN npm install -g npx
```
Comment (MKodde) [inline ci/docker/php-fpm/Dockerfile:12]: Why apt install when we can get it from the official node docker container..

This is what we did in the EngineBlock Github Actions Test integration. And it works quite well in my opinion.

File: composer.json [hunk 2]
```diff
@@ -110,7 +110,7 @@
         "behat-selenium": "./ci/qa/behat-selenium",
         "test": "./ci/qa/phpunit",
         "license-headers": "vendor/bin/docheader check src/",
-        "yarn-audit": "yarn audit --groups dependencies optionalDependencies",
+        "yarn-audit": "yarn audit",
```
Comment (MKodde) [inline composer.json:113]: Removing the audit restrictions is no problem, now that we are installing latest and greatest of the Webpack Encore packge (and its dependencies)

**Comment by phavekes** [comment]
At build time I still receive a warning on a postcss plugin:

```
[vagrant@stepup-build /vagrant]$ yarn --cache-folder=/home/vagrant/yarn_cache encore production
yarn run v1.22.11
$ /vagrant/Stepup-Gateway/node_modules/.bin/encore production
Running webpack ...

postcss-resolve-url: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
 DONE  Compiled successfully in 12929ms  
```

**Review by Badlapje — CHANGES_REQUESTED** [review]
Code-wise this looks good to me.  Given i don't have a local repo installed i refer to Peter's comment for one warning that is apparently still there.

❤️ the clarification comments btw!

**Review by Badlapje — APPROVED** [review]
I couldn't find a replacement atm for the plugin that is deprecated, so approving this as is given that deprecated still means in working order.

**Review by phavekes — APPROVED** [review]

**Pull request merged**